README.md

```diff
@@ -55,8 +55,40 @@ $ sudo mkdir /var/log/nawab
 
 * Run the script
 ```
-$ python3 nawab_bot.py
+$ python3 nawab.py
 ```
+### CLI Arguments
+
+* The bot provides the user with arguments to input and use. To view all the arguments try:
+
+```console
+$ python3 nawab.py -h
+usage: nawab.py [-r] [-b BLACKLIST] [-p PATH] [-V] [-s] [-h] [-v]
+
+optional arguments:
+  -r, --retweet         Retweet all tweets automatically, doesn't spawn a
+                        telegram bot
+  -b BLACKLIST, --blacklist BLACKLIST
+                        Blacklist the given username
+  -p PATH, --path PATH  Path where the log files be stored. Note to create
+                        directory in that path beforehand.
+  -V, --verbose
+  -s, --silent
+  -h, --help            Show this help message and exit.
+  -v, --version         Show program's version number and exit.
+
+```
+* Usage
+
+| Parameter                 | Default/value       | Description   |	
+| :------------------------ |:-------------:| :-------------|
+| -r --retweet 	       |	  boolean         |Retweet all tweets automatically, doesn't spawn telegram bot.
+|-b  --blacklist         | list           |Blacklist the given username.
+| -p --path	       |	  string          |Path where the log files be stored. Note to create directory in that path beforehand.
+| -V, --verbose 		       | 20	           | log all messages to console. 
+| -s, --silent		           | 50             | log all messages of error and above to console.
+| -h, --help	        |            | Show this help message and exit.
+|  -v, --version 	         |             | Show program's version number and exit.
```

Comment:
I hope this is still in WIP? If not, please do rephrase it. It looks similar to the above.

Also, why not give examples of usages?